frontend/appflowy_flutter/lib/plugins/base/emoji/emoji_picker.dart

```diff
@@ -24,28 +27,49 @@ class FlowyEmojiPicker extends StatefulWidget {
 }
 
 class _FlowyEmojiPickerState extends State<FlowyEmojiPicker> {
-  EmojiData? emojiData;
+  late EmojiData emojiData;
+  bool loaded = false;
+
+  void loadEmojis(EmojiData data) {
```

Comment:
Move this function below the build function. We prefer to place private functions after public functions.